core/src/main/scala/org/elasticmq/Limits.scala

```diff
@@ -68,25 +71,57 @@ object Limits {
       "ReadCountOutOfRange"
     )
 
-  def verifyMessageStringAttribute(stringAttribute: String, limits: Limits): Either[String, Unit] =
+  def verifyMessageStringAttribute(
+      attributeValue: String,
+      attributeName: String,
```

Comment:
shouldn't it be name-value, as that's the usual order? :)

Reply:
Yeah, I have changed it to name-value order :)